<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrowsLegendSettingEditWidget</class>
 <widget class="QWidget" name="ArrowsLegendSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>283</width>
        <height>436</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,1">
       <item>
        <widget class="QCheckBox" name="visibleCheckBox">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="titleGroupBox">
         <property name="title">
          <string>Title</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
          <item row="0" column="0">
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="titleEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="titleFontLabel">
            <property name="text">
             <string>Font Setting:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFontEditWidget" name="titleFontWidget" native="true"/>
          </item>
         </layout>
         <zorder>titleEdit</zorder>
         <zorder>titleLabel</zorder>
         <zorder>titleFontLabel</zorder>
         <zorder>titleFontWidget</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="lengthGroupBox">
         <property name="title">
          <string>Length</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lengthFormatEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lengthFontLabel">
            <property name="text">
             <string>Font Setting:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lengthFormatLabel">
            <property name="text">
             <string>Format:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFontEditWidget" name="lengthFontWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="colorGroupBox">
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="titleColorLabel">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="ColorEditWidget" name="titleColorWidget">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lengthColorLabel">
            <property name="text">
             <string>Length:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="ColorEditWidget" name="lengthColorWidget">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="backgroundColorLabel">
              <property name="text">
               <string>Background:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="ColorEditWidget" name="backgroundColorWidget">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="TransparencyWidget" name="backgroundOpacityWidget" native="true"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="showBorderCheckBox">
              <property name="text">
               <string>Draw border</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="imageSettingGroupBox">
         <property name="title">
          <string>Size and Position</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ImageSettingWidget" name="imageSettingWidget" native="true"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFontEditWidget</class>
   <extends>QWidget</extends>
   <header>misc/qfonteditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageSettingWidget</class>
   <extends>QWidget</extends>
   <header>guicore/image/imagesettingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
